router.go

```diff
@@ -354,6 +358,7 @@ func (app *App) registerStatic(prefix, root string, config ...Static) Router {
 		// Return request if found and not forbidden
 		status := c.fasthttp.Response.StatusCode()
 		if status != StatusNotFound && status != StatusForbidden {
+			c.fasthttp.Response.Header.Set(HeaderCacheControl, "public, max-age="+strconv.Itoa(maxAge))
```

Comment:
We should build the `CacheControl` value once outside of the handler value.
```go
maxAgeVal := "public, max-age="+strconv.Itoa(maxAge)
```

And check if the value is higher than 1 before setting it
```go
if maxAge > 0 {
  c.fasthttp.Response.Header.Set(HeaderCacheControl, maxAgeVal)
}
```

Reply:
Yea, that was my pain point at this PR. I've applied your hints